### PR TITLE
improve(reporter): Improve USDC.e refunds and outstanding transfer reporting

### PR DESCRIPTION
**Pull request by nicholaspai**
- Refunds for chains where Native USDC is not supported should not have refund entries under USDC but under USDC.e
- Outstanding transfers should split out USDC.e vs USDC

### CODE REVIEW

File: src/monitor/Monitor.ts [hunk 1]
```diff
@@ -632,22 +633,51 @@ export class Monitor {
     for (const refunds of nextBundleRefunds) {
       for (const relayer of this.monitorConfig.monitoredRelayers) {
         this.updateRelayerRefunds(refunds, relayerBalanceReport[relayer], relayer, BalanceType.NEXT);
-        this.updateCrossChainTransfers(relayer, relayerBalanceReport[relayer]);
```
Comment (nicholaspai) [inline src/monitor/Monitor.ts:635]: Not sure why this wasn't split out before but we shouldn't update xchain transfers conditional on there being a refund entry for a relayer

File: src/monitor/Monitor.ts [hunk 2]
```diff
@@ -632,22 +633,51 @@ export class Monitor {
...
-          relayer,
-          chainId,
-          l1Token.address
-        );
+        // Handle special case for USDC which has multiple L2 tokens we might hold in inventory mapped to a single
+        // L1 token.
+        if (l1Token.symbol === "USDC" && chainId !== this.clients.hubPoolClient.chainId) {
+          const bridgedUsdcAddress =
+            TOKEN_SYMBOLS_MAP[chainId === CHAIN_IDs.BASE ? "USDbC" : "USDC.e"].addresses[chainId];
+          const nativeUsdcAddress = TOKEN_SYMBOLS_MAP["_USDC"].addresses[chainId];
```
Comment (pxrl) [inline src/monitor/Monitor.ts:652]: What if there is no native USDC for the remote chain? i.e. Linea.

Comment (pxrl) [reply src/monitor/Monitor.ts:652]: Ah, I see it's handled later.

**Review by pxrl — APPROVED** [review]
Looks OK :+1:

**Pull request merged**